components/layout_2020/positioned.rs

```diff
@@ -208,7 +208,7 @@ impl<'a> AbsolutelyPositionedFragment<'a> {
             padding_border_sum: Length,
             computed_margin_start: LengthOrAuto,
             computed_margin_end: LengthOrAuto,
-            solve_margins: impl FnOnce(Length) -> (Length, Length),
+            avoid_negative_margin_start: bool,
```

Comment:
Please remove this change or introduce yet another new type for this boolean.

Reply:
Even with exactly two call sites, that both have a comment to repeat the name of the parameter?

Reply:
IIRC the collapsing margin stuff doesn't have more callers than that, and the comment can go if you introduce a type, but as I said on IRC, all my nits are optional.